pages/create/index.js

```diff
@@ -0,0 +1,243 @@
+      <PetSelection
+        onChangePet={handleChangePet}
+        animalList={animalList}
+        currentPet={currentPet}
+        name="type"
+      />
+      <StyledMainField>
+        <StyledFormElement>
+          <label>Type:</label>
+          <input name="type" value={animalList[currentPet].type} disabled />
+        </StyledFormElement>
+        <StyledFormElement>
+          <label htmlFor="name">Name:</label>
+          <input
+            name="name"
+            id="name"
+            placeholder="Samantha"
+            maxLength={30}
+            required
+          />
+        </StyledFormElement>
+        <StyledFormElement>
+          <label htmlFor="characteristic">Characteristics:</label>
+          <StyledCharacteristicsContainer>
+            <select
+              name="characteristic1"
+              id="characteristic"
+              value={characteristics.characteristic1}
+              onChange={(event) =>
+                setCharacteristics({
+                  ...characteristics,
+                  characteristic1: event.target.value,
+                })
+              }
+              required
+            >
+              <option value="" disabled>
+                Choose Nr. 1
+              </option>
+              {characteristicOptions.map((option, index) => (
+                <option
+                  key={index}
+                  disabled={
+                    characteristics.characteristic2 === option.characteristic ||
+                    characteristics.characteristic2 === option.opposite
+                  }
+                >
+                  {option.characteristic}
+                </option>
+              ))}
+            </select>
+            <select
+              name="characteristic2"
+              value={characteristics.characteristic2}
+              onChange={(event) =>
+                setCharacteristics({
+                  ...characteristics,
+                  characteristic2: event.target.value,
+                })
+              }
+            >
+              <option value="">*none*</option>
+              {characteristicOptions.map((option, index) => (
+                <option
+                  key={index}
+                  disabled={
+                    characteristics.characteristic1 === option.characteristic ||
+                    characteristics.characteristic1 === option.opposite
+                  }
+                >
+                  {option.characteristic}
+                </option>
+              ))}
+            </select>
+          </StyledCharacteristicsContainer>
+        </StyledFormElement>
+      </StyledMainField>
+      <StyledIndicatorContainer>
+        {animalList[currentPet].indicators.map((indicator, index) => (
+          <Indicator key={index} data={indicator} />
+        ))}
+      </StyledIndicatorContainer>
+      <StyledConfirmButtonContainer>
+        <StyledReturnButton type="button">
+          <Link href="/">Cancel</Link>
```

Comment:
See above. Don't put Links inside buttons. Furthermore for a Cancel button we would prefer to use a button not a Link.

Reply:
Not sure what exactly you mean here. So you would suggest using a button and then use route.push inside a separate function that is triggered upon clicking the button?

Reply:
Sorry, the other way around. Use a Link and style it accordingly. But don't wrap a Link inside a button. Use one element or the other.